{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectorAuthenticationException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/ConnectorAuthenticationException","preciseIdentifier":"s:10AWSAppflow32ConnectorAuthenticationExceptionV","text":"ConnectorAuthenticationException"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsappflow\/updateconnectorregistrationoutputerror\/connectorauthenticationexception(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/UpdateConnectorRegistrationOutputError\/connectorAuthenticationException(_:)","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectorAuthenticationException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"ConnectorAuthenticationException","preciseIdentifier":"s:10AWSAppflow32ConnectorAuthenticationExceptionV"},{"kind":"text","text":")"}],"title":"UpdateConnectorRegistrationOutputError.connectorAuthenticationException(_:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:10AWSAppflow38UpdateConnectorRegistrationOutputErrorO32connectorAuthenticationExceptionyAcA0chI0VcACmF","modules":[{"name":"AWSAppflow"}]},"hierarchy":{"paths":[["doc:\/\/AWSAppflow\/documentation\/AWSAppflow","doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/UpdateConnectorRegistrationOutputError"]]},"references":{"doc://AWSAppflow/documentation/AWSAppflow":{"role":"collection","title":"AWSAppflow","abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow","kind":"symbol","type":"topic","url":"\/documentation\/awsappflow"},"doc://AWSAppflow/documentation/AWSAppflow/ConnectorAuthenticationException":{"role":"symbol","title":"ConnectorAuthenticationException","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ConnectorAuthenticationException"}],"abstract":[{"type":"text","text":"An error occurred when authenticating with the connector endpoint."}],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/ConnectorAuthenticationException","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ConnectorAuthenticationException"}],"url":"\/documentation\/awsappflow\/connectorauthenticationexception"},"doc://AWSAppflow/documentation/AWSAppflow/UpdateConnectorRegistrationOutputError":{"role":"symbol","title":"UpdateConnectorRegistrationOutputError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"UpdateConnectorRegistrationOutputError"}],"abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/UpdateConnectorRegistrationOutputError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"UpdateConnectorRegistrationOutputError"}],"url":"\/documentation\/awsappflow\/updateconnectorregistrationoutputerror"},"doc://AWSAppflow/documentation/AWSAppflow/UpdateConnectorRegistrationOutputError/connectorAuthenticationException(_:)":{"role":"symbol","title":"UpdateConnectorRegistrationOutputError.connectorAuthenticationException(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectorAuthenticationException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"ConnectorAuthenticationException","preciseIdentifier":"s:10AWSAppflow32ConnectorAuthenticationExceptionV"},{"kind":"text","text":")"}],"abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/UpdateConnectorRegistrationOutputError\/connectorAuthenticationException(_:)","kind":"symbol","type":"topic","url":"\/documentation\/awsappflow\/updateconnectorregistrationoutputerror\/connectorauthenticationexception(_:)"}}}